src/generators.ts

```diff
@@ -47,7 +47,10 @@ export function appendClass(editContent: string, className: string, fileInfo: Fi
 }
 
 export function appendIfndef(editContent: string, id: string, fileInfo: FileInfo, config: Config) {
-    return editContent.concat("#ifndef ", id, fileInfo.eol, Syntax[config.headerType].preProcessorStyle, "define ", id, fileInfo.eol, fileInfo.eol)
+    if (config.headerType == "post2017")
+        return editContent.concat("#ifndef ", id, fileInfo.eol, "    #define ", id, fileInfo.eol, fileInfo.eol);
+    else
+        return editContent.concat("#ifndef ", id, fileInfo.eol, Syntax[config.headerType].preProcessorStyle, "define ", id, fileInfo.eol, fileInfo.eol);
```

Comment:
I think we could change `Syntax[config.headerType].preProcessorStyle` inside of src/constants.ts instead of making a special case here.
`preProcessorStyle["post2017"]` currently is "\t#", so simply replacing the tab with 4 spaces should do the trick.

Reply:
Hello,
No problem, I could not find the code for preProcessorStyle yesterday, maybe I looked at it too quickly, that's why I made a special case. 

Can you edit the preProcessorStyle or do I have to edit myself the pull request?

Reply:
I can do it, no problem.  
I'll merge the PR and make this change then.
I'll get it released sometime today.